.github/workflows/version-update.yml

```diff
@@ -20,9 +20,13 @@ jobs:
           BRANCH_NAME="${GITHUB_REF#refs/heads/}"
           PACKAGE_VERSION="${GITHUB_REF##*/}"
           echo ::set-output name=branch::${BRANCH_NAME}
-          echo ::set-output name=version::${PACKAGE_VERSION}
-          if [[ $PACKAGE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]
+          if [[ $PACKAGE_VERSION =~ ^v[0-9]+\.[0-9]+(\.[0-9]+)*$ ]]
```

Comment:
This will match versions with an arbitrary number of sections (e.g. v1.2.2.2.2.2). I'm thinking we can just match versions with 2 or 3 sections by replacing that `*` with a  `?`.

Reply:
right, good point. Updated!